sdks/python/apache_beam/io/gcp/bigquery_test.py

```diff
@@ -617,6 +618,146 @@ def test_multiple_destinations_transform(self):
+    output_table_1 = '%s%s' % (self.output_table, 1)
+    output_table_2 = '%s%s' % (self.output_table, 2)
+    schema = {'fields': [
+        {'name': 'name', 'type': 'STRING', 'mode': 'NULLABLE'},
+        {'name': 'language', 'type': 'STRING', 'mode': 'NULLABLE'}]}
+
+    additional_bq_parameters = {
+        'timePartitioning': {'type': 'DAY'},
+        'clustering': {'fields': ['language']}}
+
+    table_ref = bigquery_tools.parse_table_reference(output_table_1)
+    table_ref2 = bigquery_tools.parse_table_reference(output_table_2)
+
+    pipeline_verifiers = [
+        BigQueryTableMatcher(
+            project=self.project,
+            dataset=table_ref.datasetId,
+            table=table_ref.tableId,
+            expected_properties=additional_bq_parameters),
+        BigQueryTableMatcher(
+            project=self.project,
+            dataset=table_ref2.datasetId,
+            table=table_ref2.tableId,
+            expected_properties=additional_bq_parameters),
+        BigqueryFullResultMatcher(
+            project=self.project,
+            query="SELECT name, language FROM %s" % output_table_1,
+            data=[(d['name'], d['language'])
+                  for d in _ELEMENTS
+                  if 'language' in d]),
+        BigqueryFullResultMatcher(
+            project=self.project,
+            query="SELECT name, language FROM %s" % output_table_2,
+            data=[(d['name'], d['language'])
+                  for d in _ELEMENTS
+                  if 'language' in d])]
+
+    args = self.test_pipeline.get_full_options_as_args(
+        on_success_matcher=hc.all_of(*pipeline_verifiers),
+        experiments='use_beam_bq_sink')
+
+    with beam.Pipeline(argv=args) as p:
+      TEST_ELEMENTS = list(filter(lambda x: 'language' in x, _ELEMENTS))
+      _SIZE = len(TEST_ELEMENTS)
+      test_stream = (TestStream()
+                     .advance_watermark_to(0)
+                     .add_elements(TEST_ELEMENTS[:_SIZE/2])
+                     .advance_watermark_to(100)
+                     .add_elements(TEST_ELEMENTS[_SIZE/1:])
+                     .advance_watermark_to_infinity())
+      input = p | test_stream
+
+      _ = (input
+           | "WriteWithMultipleDests" >> beam.io.gcp.bigquery.WriteToBigQuery(
+               table=value_provider.StaticValueProvider(
+                   str, '%s:%s' % (self.project, output_table_1)),
+               schema=value_provider.StaticValueProvider(dict, schema),
+               additional_bq_parameters=additional_bq_parameters,
+               method='STREAMING_INSERTS'))
+      _ = (input
+           | "WriteWithMultipleDests2" >> beam.io.gcp.bigquery.WriteToBigQuery(
+               table=value_provider.StaticValueProvider(
+                   str, '%s:%s' % (self.project, output_table_2)),
+               schema=beam.io.gcp.bigquery.SCHEMA_AUTODETECT,
+               additional_bq_parameters=lambda _: additional_bq_parameters,
+               method='FILE_LOADS'))
```

Comment:
Try
`$RUN_CMD 2>&1 | tee $LOG_FILE && cat $LOG_FILE | grep "Not Implemented Error" `